ipaserver/install/server/install.py

```diff
@@ -855,7 +854,10 @@ def install(installer):
     service.print_msg("Applying LDAP updates")
     ds.apply_updates()
 
-    # Restart krb after configurations have been changed
+    # Restart ds and krb after configurations have been changed
+    service.print_msg("Restarting the directory server")
+    ds.restart()
```

Comment:
I am not sure, I tink this came from a change @abbra made to forward port the patch.